{{ cookiecutter.hosting_github_repo_name }}/admin-docs/updating-to-new-cookiecutter-releases.md

```diff
@@ -0,0 +1,85 @@
+# Updating to New Cookiecutter Releases
+
+As we have abstracted away the `cdp-backend` for pipeline, infrastructure,
+and, database functionality, and `cdp-frontend` for web-app functionality,
+and because every time the pipelines which utilize those packages run, the
+pipeline pulls in the latest (non-breaking) versions of each, you should
+rarely need to update anything yourself.
+
+However, in the instance that a new version of
+_this cookiecutter generated repository_ becomes available
+(as seen on our [releases page](https://github.com/CouncilDataProject/cookiecutter-cdp-deployment/releases)),
+you may want to pull in the changes as they might change default pipeline configuration,
+enable new features, add more documentation, and more.
+
+To do so, you should feel comfortable with the command line and git.
+
+### Steps to Upgrade
+
+1. Clone (or fetch and pull) your repository:
+
+    - `git clone {{ cookiecutter.hosting_github_url }}.git`
+
+    OR if you have previously cloned your repository and other changes have occurred:
+
+    - `git checkout main`
+    - `git fetch`
+    - `git pull main`
+
+2. Install `cookiecutter`:
+
+    You will need Python 3.6+ installed for this.
+
+    - `pip install cookiecutter`
+
+3. _Partially_ run `cookiecutter`:
+
+    We simply want to download the latest version of this cookiecutter template,
+    to do so:
+
+    - `cookiecutter gh:CouncilDataProject/cookiecutter-cdp-deployment`
+
+    Re-download the latest version and then when prompted for parameter input,
+    escape or exit the process.
+
+    ![screenshot of terminal after cookiecutter interrupt](./resources/cookiecutter-interrupt.png)
+
+4. Run `make update-from-cookiecutter`:
+
+    This will pull in the current cookiecutter updates as changes to the current
+    repository.
+
+    Ensure you are in the repository directory:
+
+    - `cd {{ cookiecutter.municipality_slug }}`
+
+    Then run:
+
+    - `make update-from-cookiecutter`
+
+5. Select the desired changes to commit:
+
+    In an editor or in the terminal you can now review the changes to commit.
+
+    With git in a terminal:
+
+    - `git status`
+    - `git add {some-file}`
+    - OR `git restore {some-file}`
+
```

Comment:
Just curious, but does this do a complete overwrite or is a merge conflict a possiblity? It seemed to overwrite when I used it, but just wanted to confirm

Reply:
I don't believe it is possible to get a merge conflict... What is happening here is cookiecutter is writing directly to the files so it should always be considered either an insertion or deletion on git's part.